cmd/immuclient/immuc/getcommands.go

```diff
@@ -83,11 +83,38 @@ func (i *immuc) VerifiedGetTxByID(args []string) (string, error) {
 	return PrintTx(tx.(*schema.Tx), true), nil
 }
 
+func (i *immuc) parseKeyArg(arg string) (key []byte, revision int64, err error) {
+	if i.revisionSeparator == "" {
+		// No revision separator - argument is the key
+		return []byte(arg), 0, nil
+	}
+
+	idx := strings.LastIndex(arg, i.revisionSeparator)
+	if idx < 0 {
+		// No revision separator in the argument - that's a key without revision
+		return []byte(arg), 0, nil
+	}
+
+	key = []byte(arg[:idx])
+	revisionStr := arg[idx+len(i.revisionSeparator):]
+
+	revision, err = strconv.ParseInt(revisionStr, 10, 64)
```

Comment:
probably `get key@` should return invalid format?

Reply:
`strconv.ParseInt` fails on empty string:

```
$ ./immuclient get key@
Invalid key revision number - not an integer: strconv.ParseInt: parsing "": invalid syntax
```